src/components/seller-form/seller-form.tpl.html

```diff
@@ -7,50 +7,110 @@
           <div class="row">
             <div class="col-sm-12">
               <div class="panel">
-                <form>
-                  <div class="form-group">
-                    <label for="sellerName">Seller Name</label>
-                    <input type="text" class="form-control" placeholder="Seller Name">
-                  </div>
-                  <div class="form-group">
-                    <label for="Countries">Countries</label>
-                    <div class="dropdown">
-                      <button class="btn btn-secondary dropdown-toggle" type="button" 
-                      id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                        Dropdown button
-                      </button>
-                      <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-                        <span>UK</span>
-                        <span>USA</span>
-                        <span>USA</span>
-                        <span>FR</span>
+                <form ng-submit="$ctrl.submit()" name="sellerDetail">
```

Comment:
[Medium]: Ideally way is to have click function on the button where that is needed to avoid submit getting called on pressing enter key